### ISSUE TITLE
shreddit not seeing some pip modules again

### ISSUE THREAD

**Issue by grrrrr** [x89/Shreddit#17]
So I've updated to the latest shreddit dated Sept 16 and run with the following errors

    $ python3 ./shreddit.py -c shreddit.cfg
    Traceback (most recent call last):
      File "./shreddit.py", line 8, in <module>
        from simpleconfigparser import simpleconfigparser
    ImportError: No module named 'simpleconfigparser'

but SimpleConfigParser is installed

    $ pip list | grep -i simple
    SimpleConfigParser (0.1.0)

and with python 2 as it looks like the python3 requirement has been removed

    $ ./shreddit.py -c shreddit.cfg
    Traceback (most recent call last):
      File "./shreddit.py", line 56, in <module>
        whitelist_distinguished = config.getboolean('main', 'whitelist_distinguished')
      File "/usr/lib/python2.7/ConfigParser.py", line 369, in getboolean
        if v.lower() not in self._boolean_states:
    AttributeError: 'NoneType' object has no attribute 'lower'

**Comment by x89** [comment]
Late reply @grrrrr sorry, thanks for sticking with the repo!

Are you having the same issue still? If so I'll take a look into it.

(I've been away quite a while, my apologies.)

**Comment by grrrrr** [comment]
hi @x89 unfortunately yes it's still throwing errors as I mention above

**Comment by Kline-** [comment]
@grrrrr Are you using forced HTTPS on reddit? I just installed Shreddit last night on Debian with Python3 and had similar errors. After a bit of Googling I found a thread about PRAW not supporting the forced HTTPS in the 2.x branch that Shreddit uses. Manually installed the PRAW3 beta and it works like a champ now.

See here: https://www.reddit.com/r/redditdev/comments/2gmzqe/praw_https_enabled_praw_testing_needed/.compact
